README.markdown

```diff
@@ -367,568 +367,14 @@ Or
 
 ## Reference
 
-Classes:
-
-* [firewall](#class-firewall)
-
-Types:
-
-* [firewall](#type-firewall)
-* [firewallchain](#type-firewallchain)
+Please refer to the REFERENCE.md for more information on the various Classes and Types of this module. Information regarding it's Facts is shown below.
```

Comment:
Suggested change: "For information on the classes and types, see the REFERENCE.md. For information on the facts, see below."

Reply:
Have updated the writing to what you suggested. Have also added a link the the REFERENCE.md

Reply:
Great!